app/Event/EventReadServiceProvider.php

```diff
@@ -52,5 +59,23 @@ function () use ($container): JSONLDMainLanguageQuery {
                 );
             }
         );
+
+        $container->addShared(
+            EventPlaceHistoryProjector::class,
+            function () use ($container): EventPlaceHistoryProjector {
+                return new EventPlaceHistoryProjector(
+                    $container->get(EventLocationHistoryRepository::class),
+                    $container->get('event_jsonld_cache'),
+                    LoggerFactory::create($this->getContainer(), LoggerName::forService('event_location_history'))
```

Comment:
Not sure if we need a dedicated logger for this, most of the simple HTTP requests and their command and event handling or just logged inside `web.log`